soroban-sdk/Cargo.toml

```diff
@@ -10,6 +10,8 @@ version.workspace = true
 edition = "2021"
 rust-version = "1.74"
 
+exclude = ["test_snapshots/", "tests/"]
```

Comment:
The `sprban-sdk/tests` directory doesn't exist. What are you wanting to exclude with `tests/`?

Reply:
Oh interesting. This was meant to exclude src/tests, and it seems to do that when I check `cargo package --list`. I'll update the path anyways.